test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompaction_1_IT.java

```diff
@@ -95,13 +93,6 @@ public static void beforeTests() throws Exception {
     startMiniClusterWithConfig(new ExternalCompaction1Config());
   }
 
-  @AfterEach
```

Comment:
`ExternalCompactionTestUtils.configureMiniCluster` configures the compactions services. When MAC starts it inspects these properties and starts up the required Compactors based on the config. We can remove all of this setup from the tests now, except in the cases where a specialized class is being used for a test.